public/toolkit/styles/molecules/_inputs.scss

```diff
@@ -383,3 +383,27 @@ input.icon-input-field {
   line-height: $icon-input-height;
   height: $icon-input-height;
 }
+
+// Search text field with x icon to clear
+// Javascript must be added to
+$clear-icon-margin: 13.5px;
+$clear-icon-size: 16px;
+.search {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+
+  input {
```

Comment:
i think width: 100% will automatically be applied to the input

Reply:
whoops missed this